README.md

```diff
@@ -1,12 +1,21 @@
-#This project was abandoned because [Moxie Marlinspike said he was not OK with LibreSignal using the Open Whisper Systems servers and the name "Signal"](https://github.com/LibreSignal/LibreSignal/issues/37#issuecomment-217211165).
+#This GitHub repository has been abandoned.
```

Comment:
>I really like to call it project
>Sorry for being picky.

I think it would be misleading to say that the LibreSignal project has been abandoned, because @xmikos is now distributing an app with the same name that isn't based on the code in this GitHub repository. Everything would be much clearer if @xmikos had chosen a different name for his rebranded version of Signal, but now the LibreSignal project is living on through @xmikos's F-droid repo.